lib/emoji_data/emoji_char.rb

```diff
@@ -92,6 +103,13 @@ def variant?
       @variations.length > 0
     end
 
+    # Does the `EmojiChar` have an alternate skin variant encoding?
+    #
+    # @return [Boolean]
+    def skin_variant?
```

Comment:
Since the number of possible skin variations is constant (either N>1 or 0), this method name should probably be plural, e.g. `skin_variations?`.